README.asciidoc

```diff
@@ -212,19 +212,21 @@ NOTE: You can use the link:https://github.com/mikery/vagrant-ami[Vagrant-AMI] pl
 
 Install plugin from rubygems:
 ----
-$ vagrant plugin install vagrant-openstack-plugin
+$ NOKOGIRI_USE_SYSTEM_LIBRARIES=1 vagrant plugin install vagrant-openstack-plugin
```

Comment:
I don't think everyone needs to do this.  This is generally a workaround to installing patch or other packages so the build will work.